### ISSUE TITLE
README enhancement for tslab install and --python option

### ISSUE THREAD

**Issue by TonyGravagno** [yunabe/tslab#93]
Install instructions in README:  

> tslab install [--python=python3]  
> By default, tslab is registered with python3 in unix-like system and python in Windows. If Jupyter is installed with a different Python in your system, please specify the python command with --python flag.

I already have Python3 installed and the python.exe is version 3, so there isn't a need to specify the (yes, optional) python command.  
This isn't clear in that instruction.  
So, after checking `python --version`, it seems `tslab install` and no option did install correctly.

Suggested enhancement:

> By default, tslab is registered with python3 in unix-like (Linux) systems and python in Windows. On a unix-like (Linux) system, if `python3 --version` returns a version, there is no need to specify a python version. On Windows, if `python --version` returns a version, there is no need to specify a python version. The `python` command may already be a synonym for Python v3. If Jupyter is installed with a different synonym/alias to execute Python in your system, specify the python command with --python flag.

**Comment by gnaaruag** [comment]
yup a simple `tslab install` as opposed to the `tslab install [--python=python3]` seems to do the trick. A bit confusing, a fix to that would be nice.
